src/review/views.py

```diff
@@ -1050,6 +1050,15 @@ def add_review_assignment(request, article_id):
     """
     article = get_object_or_404(submission_models.Article, pk=article_id)
     reviewers = logic.get_reviewer_candidates(article, request.user)
+
+    # if setting enabled, fetch reviewers who have completed a review
+    # in a past review round.
+    past_reviewers = []
```

Comment:
Should we pass `past_reviewers` to get_reviewer_candidates either to get them highlighted (e.g with an annotation) or at least filtered out from the second table?